internal/kubernetes/operator/dbusers/dbusers_test.go

```diff
@@ -213,7 +221,15 @@ func TestBuildDBUsers(t *testing.T) {
 		}
 
 		if !reflect.DeepEqual(users[0], expectedUser) {
-			t.Fatalf("User result doesn't match.\r\nexpected: %v,\r\ngot: %v\r\n", expectedUser, users[0])
+			ed, err := json.MarshalIndent(expectedUser, "", " ")
+			if err != nil {
+				panic(err)
+			}
+			gd, err := json.MarshalIndent(users[0], "", " ")
+			if err != nil {
+				panic(err)
```

Comment:
any reason to use panic instead of t.Fatal?

Reply:
Fixed